Dockerfile

```diff
@@ -1,5 +1,10 @@
-FROM --platform=$BUILDPLATFORM cgr.dev/chainguard/wolfi-base as build
-RUN apk update && apk add build-base git openssh go-1.20
+FROM --platform=$BUILDPLATFORM golang:1.23-alpine as build
+
+WORKDIR /work
+
+COPY convex_client/ convex_client/
+WORKDIR /work/convex_client
+RUN go mod download
```

Comment:
I'm not really convinced this is correct? Having to manually run `go mod download` for a dependency seems like something is wrong. 
I'm not familiar with go modules either, so maybe check with ChatGPT or just look at how this package's other  dependencies are packaged on GitHub. 

regardless, there's no need to do this extra `COPY` and `WORKDIR` -- we're already running `COPY . .` on line 14, so everything is being copied into `/work`.